paimon-core/src/main/java/org/apache/paimon/WriteMode.java

```diff
@@ -32,7 +32,11 @@ public enum WriteMode implements DescribedEnum {
             "append-only",
             "The table can only accept append-only insert operations. Neither data deduplication nor any "
                     + "primary key constraints will be done when inserting rows into paimon."),
-    CHANGE_LOG("change-log", "The table can accept insert/delete/update operations.");
+    CHANGE_LOG("change-log", "The table can accept insert/delete/update operations."),
+    TABLE(
+            "table",
+            "The table without bucket.  Don't guarantee the order in the table. "
```

Comment:
We can have streaming read too. But just no order guarantee.
`This table is used in the traditional sense of "table". It can act like a Hive or Iceberg table. It can be written and read in a distributed manner without the concept of a bucket. It can be streaming read but no order guarantee.`